### DISCUSSION TITLE
[Dynamic Links] Firebase Dynamic Links Offline

### DISCUSSION THREAD

**Discussion by martyfuhry** [firebase/flutterfire#5580] [Feature request]
**Is your feature request related to a problem? Please describe.**
I'd like non-shortened Firebase Dynamic Links to properly be handled while the device is offline. Turning my phone on airplane mode and clicking a Firebase Dynamic Link opens my app. But the onLink method is never called with the link's parameters. While I'm aware shortened Firebase Dynamic Link URLs need a network connection, I'd like to be able to handle non-shortened Firebase Dynamic Link URLs in my app while offline.

**Describe the solution you'd like**
When I have no network connection and tap a non-shortened Firebase Dynamic Link URL, the app should open and the link should be handled by the onLink method, or by the getInitialLink method. I would expect to be able to retrieve the link parameters from the Firebase Dynamic Link which opened the app.

**Additional context**
Again, while offline, the links will properly open the app, but onLink is never called, and getInitialLink returns null.

**Comment by LucasTejero** [comment]
Im having the same problem but i have wifi,i dont know why.
